ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-eks-lifecycle/SKILL.md

```
@@ -0,0 +1,42 @@
+bash "${CLAUDE_SKILL_DIR}/scripts/check-eks-lifecycle.sh" \
+  --mapt-ref ci-operator/step-registry/redhat-developer/rhdh/eks/mapt/create/redhat-developer-rhdh-eks-mapt-create-ref.yaml \
+  --test-pattern "^e2e-eks-"
+```
+
+The script queries two sources:
+- **Primary**: `awsdocs/amazon-eks-user-guide` raw AsciiDoc on GitHub — official AWS EKS docs source with standard/extended support status and release calendar
+- **Cross-verify**: `https://endoflife.date/api/amazon-eks.json` — community-maintained EOL dates
+
+2. If the API call fails, fall back to the vendor docs:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)